PWGGA/Hyperon/AliSigma0PhotonMotherCuts.h

```diff
@@ -129,13 +129,13 @@ class AliSigma0PhotonMotherCuts : public TObject {
   int fPDGDaughter1;   //
   int fPDGDaughter2;   //
 
-  float fMassSigma;         //
-  float fSigmaMassCut;      //
-  float fSidebandCutUp;     //
-  float fSidebandCutDown;   //
-  float fPhotonPtMin;       //
-  float fPhotonPtMax;       //
-  float fRapidityMax;       //
```

Comment:
not so nice to just commit empty spaces due to the formatting

Reply:
I ran the beautifier, should I correct it by hand?

Reply:
Please

Reply:
Can you fix this please